bee-common-derive/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 IOTA Stiftung
+// Copyright 2020-2021 IOTA Stiftung
```

Comment:
I am sorry, maybe one of my comments was misinterpreted, but it should be 2021 only :(

Reply:
[`sd`](https://github.com/chmln/sd) is my best friend
```bash
sd '2020-2021' '2020' $(fd --type file)
```

Reply:
Well... you replaced with 2020 :trollface:

Reply:
```bash
sd '2020' '2021' $(fd --type file)
```